index.html

```diff
@@ -187,42 +187,64 @@
       
       <div class="slideshow-container">
       
-      <div class="mySlides" style="display: block;">
-        <h3><b> <center>Please pardon our mess, this site is still undergoing construction</center></b><br> <center> We're working diligently to bring you a better experience. </center> </h3>
+        <div class="mySlides"> 
+          <h3><b><center>Our Hearts and Minds go out to those devastated by the tornado that ravaged Little Rock.</center></b><br><center>The city of Little Rock has shelters available to those in need</center></h3>
+          <p class="author">-Main Exploration Bot v.1.17-</p>
+        </div>
+
+        <div class="mySlides">
+          <h3><b><center> If you would like volunteer to aid with cleanup please visit  </center></b><br><center>http://littlerock.gov/volunteer</center></h3>
+          <p class="author">-Main Exploration Bot v.1.17-</p>
+        </div>
+     
+      <div class="mySlides"> 
+        <h3><b><center></center></b><br><center>If you would like to donate to help with recovery efforts, text "GIVE" (without quotes) to 501 - 378 - 1111</center></h3>

```

Comment:
> If you would like to donate to help with recovery efforts, text "GIVE" (without quotes) to 501 - 378 - 1111

Where did you find this phone number? I can't find anywhere online that says this.

Reply:
That was a typo, it was supposed to be the text number for thv11 : 501 376 111

Reply:
Correct